### PR TITLE
Updated L01 typos and computers-are-fast citation.

### PR DESCRIPTION
**Pull request by bhamodi**
Changelog:
- Updated `computers-are-fast` citation to use their https site.
- Fixed numerous small grammatical typos in L01.

### CODE REVIEW

**Comment by patricklam** [comment]
Thanks for the suggestions. Part of good copy editing is leaving in author idiosyncracies that are not incorrect. e.g. "you and a team" is fine and, I'd argue, rhetorically better.

File: lectures/459.bib [hunk 1]
```diff
@@ -35,15 +35,15 @@ @book{osc
  year = {2013},
  isbn = {978-1118063330},
  publisher = {John Wiley \& Sons},
-} 
+}
```
[the remark this answers is not in the document]
Comment (bhamodi) [reply lectures/459.bib]: Sorry about that, my IDE does that automatically on any file I edit. Let me know if you require me to revert these changes.

[the remark this answers is not in the document]
Comment (jzarnett) [reply lectures/459.bib]: No, I wouldn't require you to revert it or split it into separate commits before merging. I was just making a note for the future.

**Comment by bhamodi** [comment]
Alright, reverted the "you and a team" sentence and squashed my commits. Should be good to go now.

**Pull request closed without merging**